packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -232,15 +381,161 @@ const PreviousDateRangePickerImpl = ({
+										showingTime ? undefined : '6'
+									}
+									borderBottomRightRadius={
+										showingTime ? undefined : '6'
+									}
+									pl={'6'}
+								>
+									<Text size="small">
+										{selectedDates[0]
+											? formatDisplayedDate(
+													selectedDates[0],
+											  )
+											: 'Start Date'}
+									</Text>
+								</Box>
+								{showingTime ? (
+									<Box
+										border={
+											startTimeIsValid
+												? 'secondary'
+												: 'error'
+										}
+										borderTop={
+											startTimeIsValid ? 'none' : 'error'
+										}
+										borderBottomLeftRadius={'6'}
+										borderBottomRightRadius={'6'}
+									>
+										<Form.Input
+											name={'startTime'}
+											placeholder={startTimePlaceholder}
+											type="input"
+											color={'n8'}
+											style={{
+												boxSizing: 'border-box',
+												border: 'none',
+												background: 'none',
+											}}
+											onChange={(event) =>
+												handleTimeChange(event, 'start')
+											}
+										/>
+									</Box>
+								) : null}
+							</Box>
+							<Box style={{ width: 116 }}>
+								<Box
+									border={'secondary'}
+									py={'8'}
+									pl={'6'}
+									borderBottom={
+										endTimeIsValid ? 'secondary' : 'none'
+									}
+									borderTopLeftRadius={'6'}
+									borderTopRightRadius={'6'}
+									borderBottomLeftRadius={
+										showingTime ? undefined : '6'
+									}
+									borderBottomRightRadius={
+										showingTime ? undefined : '6'
+									}
+								>
+									<Text size="small">
+										{selectedDates[1]
+											? formatDisplayedDate(
+													selectedDates[1],
+											  )
+											: 'End Date'}
+									</Text>
+								</Box>
+								{showingTime ? (
+									<Box
+										border={
+											endTimeIsValid
+												? 'secondary'
+												: 'error'
+										}
+										borderTop={
+											endTimeIsValid ? 'none' : 'error'
+										}
+										pl={'6'}
+										borderBottomLeftRadius={'6'}
+										borderBottomRightRadius={'6'}
+									>
+										<Form.Input
+											name={'endTime'}
+											placeholder={endTimePlaceholder}
+											type="input"
+											color={'n8'}
+											style={{
+												boxSizing: 'border-box',
+												border: 'none',
+												background: 'none',
+											}}
+											onChange={(event) =>
+												handleTimeChange(event, 'end')
+											}
+										/>
+									</Box>
+								) : null}
+							</Box>
+							<Box
+								border={'divider'}
+								borderRadius={'4'}
+								as="button"
+								p={'7'}
+								cursor="pointer"
+								background={'n4'}
+								style={{
+									width: 28,
+									height: 28,
+									background: showingTime
+										? colors.p9
+										: colors.n4,
+								}}
+								onClick={handleShowingTimeToggle}
+							>
+								<IconSolidClock
```

Comment:
icon is off-center

![image](https://user-images.githubusercontent.com/1351531/234167831-e6bcc524-d143-45c5-9268-ec84e99a52ae.png)

Reply:
Thank you. Fixed the positioning and now it's centered.

Reply:
looks great now!